### ISSUE TITLE
No matches for $.viewer.message_threads.sync_sequence_id

### ISSUE THREAD

**Issue by web36m** [jgeboski/purple-facebook#179]
Yesterday everything worked fine (2015-12-14). Today the problem (2015-12-15).
```
No matches for $ .view.message_threads.sync_sequence_id
```
```
$ pidgin -v
Pidgin 2.10.11 (libpurple 2.10.11)
```
```
$ lsb_release 
No LSB modules are available.
shcoder@x:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 15.10
Release:	15.10
Codename:	wily
```
```
$ uname -a
Linux x 4.2.0-19-generic #23-Ubuntu SMP Wed Nov 11 11:39:30 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by web36m** [comment]
Installed as described herein https://github.com/jgeboski/purple-facebook/wiki/Building-on-*NIX#from-the-git-repository . Problem still exists.

**Comment by web36m** [comment]
Now it works. Apparently this is a bug Facebook.

**Comment by jgeboski** [comment]
Is it working now?

**Comment by jkufner** [comment]
I noticed this too, using Spectrum transport on Debian. It went away on its own and chat is working (tested few hours ago).

**Comment by web36m** [comment]
@jgeboski Since yesterday the error was not.

**Comment by jgeboski** [comment]
Any status updates on this issue?

**Comment by dequis** [comment]
I think we can assume it was just a temporary server side issue

**Comment by jgeboski** [comment]
Yup. Without this value, the plugin will not work at all. If this issue issue still exists, please reopen.

**Comment by doctorcolossus** [comment]
I have been getting this error all day today...
`No matches for $.viewer.message_threads.sync_sequence_id`
I can't reopen, since it isn't my bug, so I hope that this comment reaches you.

**Comment by ge0rgecz** [comment]
Same problem here, today pidgin returns "No matches for $.viewer.message_threads.sync_sequence_id"

ver: Ubuntu 15.10, Pidgin 2.10.11 (libpurple 2.10.11)

**Comment by jgeboski** [comment]
@doctorcolossus @ge0rgecz: does it ever resolve itself, or does it fail indefinitely? I am thinking we might need to update the query hashes, as the old ones are from six or seven months ago.

**Comment by doctorcolossus** [comment]
It stayed broken most of yesterday, but now it's connecting properly again for me.

**Comment by jgeboski** [comment]
If someone could send me a [debugging log](https://github.com/jgeboski/purple-facebook/wiki/Debugging-Logs) of this happening, that would be great. This is likely going to require that the plugin switch to an updated revision of the Facebook Messenger protocol.

**Comment by doctorcolossus** [comment]
Unfortunately I'm no longer able to reproduce the problem for the time being, but if/when it recurs, then sure - I will try to capture a debug log.

**Comment by ge0rgecz** [comment]
"No matches for $.viewer.message_threads.sync_sequence_id" error disapeared next day. Its ok now.

**Comment by Riamse** [comment]
It's happening again on my end but judging frm the rest of this thread I hope that it disappears by tonight.

**Comment by Riamse** [comment]
Yeah it fixed itself

**Comment by XplizitBe** [comment]
It happend to me this morning at 9am and it's still giving me the error while trying to connect from bitlbee. Everything worked fine till today, when I got disconnected and now I can't connect no more.

**Comment by samadivk** [comment]
I can confirm that I currently have this issue with bitlbee as well.
I'm hoping this will clear up, as the plugin works perfectly otherwise.

**Comment by XplizitBe** [comment]
And now it works again. It reconnected by itself. Hopefully this doesn't happen all to often. I love Bitlbee for chatting.

**Comment by monsieurmax** [comment]
Happened to me today for the first time, disappeared as soon as I logged on facebook website.
I did not connect on Facebook website prior to that, but I used Franz.

**Comment by dl748** [comment]
Looks to be a problem with either Pidgin or the Plugin.  Once I got the error, reconnecting always returned this error.  Restarting Pidgin, allowed me to log.

**Comment by dequis** [comment]
**If anyone happens to be reading this while experiencing this issue: please test the android messenger app as soon as possible, and report if it works.**

I'm almost completely certain this is just a server-side issue that fixes itself after a few minutes most of the time. I don't think I can do much other than improving the error message to be clearer.

**Comment by vdmkenny** [comment]
I'm experiencing this right now. No matches for $.viewer.message_threads.sync_sequence_id
Unfortunately I do not use the android app so I can't test it.

**Comment by 7hibault** [comment]
I'm experiencing this right now, and the android messenger app works fine @dequis

**Comment by sinzim** [comment]
I've been seeing this for a day now.

**Comment by dequis** [comment]
@sinzim Go to #306
